config/database.yml

```diff
@@ -34,6 +38,8 @@ default: &default
   host: <%= host %>
   database: <%= database %>
   port: <%= port %>
+  sslmode: <%= sslmode || "disable" %>
```

Comment:
Can we use the same default of `prefer` also here please? :slightly_smiling_face:

Reply:
changed!